rotkehlchen/tests/unit/test_globaldb_upgrades.py

```diff
@@ -23,16 +23,14 @@ def _old_strethaddress_to_identifier(address: str) -> str:
 def _count_v2_v3_assets_inserted() -> int:
     """Counts and returns how many assets are to be inserted by globaldb_v2_v3_assets.sql"""
     assets_inserted_by_update = 0
-    start_counting = False
     dir_path = Path(__file__).resolve().parent.parent.parent
     with open(dir_path / 'data' / 'globaldb_v2_v3_assets.sql', 'r') as f:
         line = ' '
         while line:
             line = f.readline()
-            if start_counting:
-                assets_inserted_by_update += 1
-            if 'INSERT INTO assets' in line:
-                start_counting = True
+            assets_inserted_by_update += 1
+            if ';' in line:
+                break
```

Comment:
Why this? The reges allows multiline inserts

Reply:
Not sure that I got you. This method is intended to work only with `globaldb_v2_v3_assets.sql` which is not supposed to be changing, so I guess any simple logic should be fine since we don't need the able to handle different formats.

Reply:
Why did you make this change? I don't understand.